osu.Game/Storyboards/StoryboardSprite.cs

```diff
@@ -209,24 +203,104 @@ private interface IGeneratedCommand
             public double StartTime => command.StartTime;
 
             private readonly DrawablePropertyInitializer<T>? initializeProperty;
-            private readonly DrawableTransformer<T> transform;
             private readonly CommandTimeline<T>.TypedCommand command;
 
-            public GeneratedCommand(CommandTimeline<T>.TypedCommand command, DrawablePropertyInitializer<T>? initializeProperty, DrawableTransformer<T> transform)
+            public GeneratedCommand(CommandTimeline<T>.TypedCommand command, DrawablePropertyInitializer<T>? initializeProperty)
             {
                 this.command = command;
                 this.initializeProperty = initializeProperty;
-                this.transform = transform;
             }
 
             public void ApplyTo(Drawable drawable)
             {
                 initializeProperty?.Invoke(drawable, command.StartValue);
 
-                using (drawable.BeginAbsoluteSequence(command.StartTime))
+                switch (command.PropertyName)
```

Comment:
> Some transforms are considering the specified duration, and some are being applied instantaneously with a delay between start and end time

Transforms with delay in-between are the case of special commands handling ([Parameter Commands](https://osu.ppy.sh/wiki/en/Storyboard/Scripting/Commands#parameter-(p)-command)), where the value can't be adjusted over time, but is only active during the specified time duration.

Reply:
Sure, I suppose this improves things because there's no longer any need to create a transform for such properties. 

In that case, why not just define a bool parameter `IsParameterCommand` and do the special treatment based on that, while keeping the transforms defined by each property just like it was in master?

Also, this doesn't explain why `VectorScale` is handled on a separate case statement from `default`, despite looking like it does the same transform?